### ISSUE TITLE
Create strategy to delete harmonisation log files 

### ISSUE THREAD

**Issue by ljwh2** [EBISPOT/goci#1349]

**Comment by jiyue1214** [comment]
I have cleaned log files older than 1 month. The new cleaning script is running daily to clean the day job launched 1 month ago.

**Comment by jiyue1214** [comment]
For jobs finished **before** 19th June 2024,
- I have generated the error logs in one file and storage the file on the /lts
- Cleaned the logs older than 1 month

For jobs run **after** the 19th of June 2024:
It will be cleaned after 1 month, which is managed by a scrontab job running daily (a small and quick job)

For the log_clean_script:
1. Added into the scrontab joband will run daily 19th July.
2. updated to the gwasutil repo

How large the log file will be:
1. 700K including 20230109 to yesterday.
2. Daily log will be around 20K.

Error message example:
20240621 majordirectionmaptobuild 1 GCST90133175 raise KeyError(key) from err;KeyError: 'variant_id';
q
20240621 majordirectionmaptobuild 1 GCST90264796_chr8 value = float(value);ValueError: could not convert string to float: '+';
